src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
```

Comment:
We should not use double elements in BEM class names. In this case, we have "navigation__items-list__item", which is incorrect. The correct construction should look more or less like this: "navigation__item".

Read more about BEM methodology (with great examples) here: [CSS Tricks | Learn BEM](https://css-tricks.com/bem-101/)